options/options.html

```diff
@@ -6,13 +6,28 @@
 
 <body>
   <label>
-    <input type="number" placeholder="32" id="MB">
-    Maximum available decoding memory in Megabytes (MB)
+    <select id="DecoderMemoryLimitInMegabytes">
+      <option value=2>2 MB</option>
+      <option value=4>4 MB</option>
+      <option value=8>8 MB</option>
+      <option value=16>16 MB</option>
+      <option value=32>32 MB</option>
+      <option value=64>64 MB</option>
+      <option value=128>128 MB</option>
+      <option value=256>256 MB</option>
+    </select>
+    Maximum available decoding memory in Megabytes
+  </label>
+  <label>
+    <BR>
+    <!-- Must restart due to loading of background.js only once -->
+    <input type="checkbox" id="DebugOutput" title="Browser must be restarted for this to take effect.">
```

Comment:
I'm assuming it has to do with the way Chrome reloads "unpacked extensions", but when testing Chrome resets ShowDebugOutput after reload.  It works "fine" in Firefox except for an apparent bug that throws ["ReferenceError: chrome is not defined[Learn More]"](https://bugzilla.mozilla.org/show_bug.cgi?id=1349558) so the add-ons manager needs manually refreshed.